Utilities/bootstrap

```diff
@@ -584,6 +584,7 @@ def build_swiftpm_with_cmake(args):
         "-DSwiftCrypto_DIR="      + os.path.join(args.build_dirs["swift-crypto"],          "cmake/modules"),
         "-DSwiftSystem_DIR="      + os.path.join(args.build_dirs["swift-system"],          "cmake/modules"),
         "-DSwiftCollections_DIR=" + os.path.join(args.build_dirs["swift-collections"],     "cmake/modules"),
+        "--target", "swift-bootstrap",
```

Comment:
This doesn't seem to actually work, I can see it in the invocation, but other targets are still being built, e.g. the other executables.

Reply:
Ah, since we don't actually build using CMake, we need to do the target filtering via `ninja`.